array_api_compat/common/_helpers.py

```diff
@@ -55,9 +55,9 @@ def is_jax_array(x):
     if 'jax' not in sys.modules:
         return False
 
-    import jax.numpy
+    import jax
 
-    return isinstance(x, jax.numpy.ndarray)
+    return isinstance(x, jax.Array)
```

Comment:
Should this be more guarded? e.g. what if someone has a module named `jax.py` in their path, it could lead to an `AttributeError` here and make `array_api_compat` unusable.

Reply:
I don't know. That sort of thing tends to just break everything anyway. I've never really felt that libraries should protect against that sort of thing. 

Anyway, the whole point of this function is to be guarded. It won't import `jax` unless it's already been imported, because jax is a slow import and unnecessary if a user is using another array library.

Reply:
More precisely, it won't import `jax` unless *something* named `jax` has been imported, and then `array_api_compat` will irrecoverably error if that `jax` isn't the one we're expecting.

Same for every other package name referenced in this module.

My feeling is: it costs virtually nothing to wrap this all in an appropriate `try ... except`, so why not do so?

Reply:
I'm still not so sure this is a good idea. Usually if you have that sort of thing it will be an error for a lot of things, not just array_api_compat. My worry here is that guarding isn't as straightforward as it might seem. Wrapping everything in try/except could mean we end up silencing legitimate errors.

Reply:
OK, sounds good.